src/pages/about.astro

```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
       <section>
         <h2>What payments are eligible?</h2>
 
-        <p
-          >Companies directly pay maintainers of Open Source projects they depend on, with no strings attached. Paying
-          developers to do specific work does not count towards the Pledge. Feel free to participate in <a
-          href="https://github.com/opensourcepledge/osspledge.com/issues/112">our ongoing discussion</a> on what counts
-          and what doesn't.</p
-        >
+        <p>The primary intent with the Pledge is <b>no-strings-attached payments
+        to community projects</b>. Start there.</p>
+
+        <ol>
+
+          <li>Open Source projects that your company controls are out of scope.
+            If you own the trademark, it doesn't count.</li>
+
+          <li>Open Source projects that exclusively benefit your company or
+            your company's own ecosystem are out of scope.</li>
+
+          <li>It's okay if you get non-exclusive logo placement or other
+            acknowledgement for your payments, or an hour or two of consulting
+            time with the project team, but anything beyond such minimal
+            benefits puts the payment out of scope.</li>
+
+          <li>Payments to foundations are generally in scope, though most
+            foundations today are not set up to pay maintainers (they focus
+            more on conferences, trademarks, etc.). We hope that the Pledge
+            will enable foundations to shift towards paying maintainers in the
+            future.</li>
```

Comment:
This is contradictory to `no-strings-attached payments to community projects` – I would reword this to explain why foundations are anyway acceptable

I would mention something about the non-profit / for-profit / non-commercial aspects of foundations @tieguy mentioned – else different people might interpret "foundation" differently, some thinking it refers to non-profits only, other thinking it includes wider support

Reply:
> This is contradictory to "no-strings-attached payments to community projects"

Fair enough, resolved in bd7928b304412eff4cae823be3c3bf79f122b50c by shifting focus back to maintainers up above.

> different people might interpret "foundation" differently

Rather than clutter the language here further, I've brought back the link to [fossfoundation.info](https://fossfoundation.info/) in 
a0fc6359ecedee38a0a04c7f161605e91c9e9821. I'm with @ShaneCurcuru at https://github.com/opensourcepledge/opensourcepledge.com/issues/112#issuecomment-2395038843 that we should let the Pledge get established first before pushing too hard on the foundation question. I think it's enough that we briefly acknowledge just a bit of the complexity here.